ui/imports/utils/Constants.qml

```diff
@@ -1012,6 +1012,8 @@ QtObject {
     readonly property string statusHelpLinkPrefix: `https://status.app/help/`
     readonly property string downloadLink: "https://status.im/get"
     readonly property string sendViaChatPrefix: '//send-via-personal-chat//'
+    readonly property string statusDiscussPageUrl: 'https://discuss.status.app/c/features/51'
```

Comment:
`statusDiscussPageUrl` sounds like a generic page, eventhough it points to a very specific one (Feature requests)

Reply:
It's not the link, but rather the name of the property that sounds too generic

Reply:
I can change to statusFeatureRequestUrl :)

Reply:
you are right perhaps should point to this instead `https://discuss.status.app`